core/src/main/java/org/elasticsearch/index/fielddata/IndexFieldDataService.java

```diff
@@ -108,7 +110,20 @@ public synchronized void clearField(final String fieldName) {
     @SuppressWarnings("unchecked")
```

Comment:
I'm pretty sure this is actually referring to the `(IFD)` cast at the return statement of this method, so this shouldn't be removed.